alts/shared/models.py

```diff
@@ -389,3 +389,6 @@ class SchedulerConfig(CeleryConfig):
     working_directory: str = '/srv/alts/scheduler'
     jwt_secret: str
     hashing_algorithm: str = 'HS256'
+    excluded_pkgs_url: str = 'https://git.almalinux.org/almalinux/alts-exclusions/raw/branch/main/skipped_tests.json'
```

Comment:
I'd probably rename this variable and the other related ones to something more meaningful. For instance, `excluded_tests_url`, `excluded_tests_cache_size` and `excluded_tests_cache_update_interval`. Yes, they're longer, but they leave no option for confusion.